### DISCUSSION TITLE
invisible/wrong texture cosmetics and maybe something irrelevant/misplaced

### DISCUSSION THREAD

**Discussion by superon** [apace100/cosmetic-armor#10] [Q&A]
o/'
i added a few things to a cosmetic armor slot (1.16.5) which seem to be invisible:
heads and skulls (player heads, dragons head, skeleton skull etc)
and a thing i notice is apparently thinking it should look like an iron helm (goggles from goml) though they very much don't look like that when placed in the regular helm slot (trying harder at this, it turns out they do look like a regular iron helmet).

so question: is there a way to resolve these and if so how?

also:
as i wish to have everything otherwise wearable available to be worn in the cosmetic slots there seems little point (beyond debugging) to having the "slot: [slot]" tooltip so wondering if you'd know off hand how to remove that?
(i feel like this latter query maybe not in the realms of cosmetic armour but curios itself, no doubt it has been answered there but i've not searched hard enough just yet)

**Comment by superon** [comment]
oh lordy, i must apologise... sorry, i didn't see the issues page over there where these already appear to be very answered :/
well pretty very answered, still unsure about cosmetics for things like cowboy hat from biomemakeover and goggle from goml..

edit to add that i've read the response on player heads and i'm not wanting an update, just asking if you know if there's a way to redirect the cosmetic to output an image, bc as i mention the goggles appear to be a regular iron helm if placed in the cosmetic slot

**Reply by apace100** [reply]
Sorry, I'm not really sure what you mean by that. Can you provide a screenshot?
Either way, the 1.16 version is not very flexible in terms of accepting items (only tag-based) or modifying appearances (which you seem to be wanting to do, somehow?) and I'm probably not gonna spend any development time on it to make it possible. But I'd still like to know what you're trying to do, so I can make sure this is possible in the newer versions, at the very least.

**Reply by superon** [reply]
ok, i guess what i'm getting at here is:
is there any way you can add the ability for pack devs to add in their own resources to appear in the custom armor slots?

i imagine this would be a simple script file and folder that can contain pngs (resource pack style setup perhaps or maybe something simpler;) for adding simple cube designs (for the head for example) or perhaps even including model files for additional optional angles.
extending on this, a template explaining where the surfaces are mapped would be useful.

this way i can add an approximation of a creeper head (for example) or a ring of flowers a headband a tennis cap or whatever :)
also, idk if this is already a thing, i think it would be cool to have the option to add the custom experience to items added into trinket/curio slots... not sure if that final thing could exist comfortably within the scope of "custom armor" of course.

hope that at least makes more sense :) idk how easy/difficult any of this maybe and of course feel free to tell me i'm spouting nonsense..

note: i just thought i could well have misunderstood the intention here as you asked for a screenshot :P 
i think i can explain better if i try again,  the goggles from goml just redirect the texture resource to minecraft iron helm texture.    this is something the mod does in purpose and most likely intended so nothing wrong with that behaviour particularly when it comes to mod integration although i would prefer the look different.
